mne/io/edf/tests/test_edf.py

```diff
@@ -52,8 +53,8 @@ def test_concat():
 
 def test_bdf_data():
     """Test reading raw bdf files"""
-    raw_py = read_raw_edf(bdf_path, montage=montage_path, eog=eog,
-                          misc=misc, preload=True)
+    raw_py = _test_raw_object(read_raw_edf, True, input_fname=bdf_path,
```

Comment:
It's actually a boolean telling whether not preloading is implemented for the reader. So basically if it's true both cases are tested.